CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Upcoming
+
+* Add possibility of adding an id to the for attribute of the label via label-it attribute
```

Comment:
The attribute is `field-id`. We are looking at `attrs.fieldId`. Also, could you add backticks please?